lib/matplotlib/tests/test_pyplot.py

```diff
@@ -367,3 +368,23 @@ def test_set_current_axes_on_subfigure():
     assert plt.gca() != ax
     plt.sca(ax)
     assert plt.gca() == ax
+
+
+def test_pylab_intergation():
+    pytest.importorskip("IPython")
+    subprocess.run(
+        [
+            sys.executable,
+            "-m",
+            "IPython",
+            "--pylab",
+            "-c",
+            "import matplotlib.pyplot as plt; assert plt._REPL_DISPLAYHOOK == plt._ReplDisplayHook.IPYTHON",  # noqa
```

Comment:
I would just implictly join the strings here to split this over two lines
```python
        [
            sys.executable,
            "-mIPython",
            "--pylab",
            "-c",
            "import matplotlib.pyplot as plt; "
            "assert plt._REPL_DISPLAYHOOK == plt._ReplDisplayHook.IPYTHON",
        ],
```

Reply:
The importance of the lack of the trailing comma is too subtle, went with a `';'.join(...)` instead.